targets/particle-argon.json

```diff
@@ -0,0 +1,6 @@
+{
+	"inherits": ["particle-3rd-gen"],
+	"build-tags": ["particle_argon"],
+	"flash-method": "openocd",
+	"openocd-interface": "cmsis-dap"
```

Comment:
Strictly speaking, these two lines are optional because they are already inherited from the `particle-3rd-gen` target. But you could keep them here too if you think that's easier to read.